docs/source/features.md

```diff
@@ -140,7 +140,29 @@ newtype Identity a = MkIdentity{runIdentity :: a}
 newtype Equal a = MkEqual{pair :: (a, a)}
 ```
 
-_Note: Unfortunately, Agda does not allow the constructor name to be the same as the data/record name._
+Unfortunately, Agda does not allow the constructor name to be the same as the data/record name.
+However, it _is_ possible to achieve this with Agda2HS if you do not explicitly add a constructor name to a record definition (though the notation on the Agda side will be a bit awkward):
```

Comment:
"the notation on the Agda side will be a bit awkward" seems a bit too subjective to include in a user manual, perhaps instead we could say "This requires use of the `record { ... }` syntax to construct an element on the Agda side" or something similar.

Reply:
Good point, I've changed it to "(this requires the use of `record { ... }` syntax on the Agda side)".